src/manifest-converter.ts

```diff
@@ -66,7 +80,10 @@ export function writeJson(json: any, ...pathPieces: string[]) {
  * Given a bower.json manifest, generate a package.json manifest for npm.
  */
 export function generatePackageJson(
-    bowerJson: any, npmName: string, npmVersion: string) {
+    bowerJson: any,
+    npmName: string,
+    npmVersion: string,
+    preferLocal?: Map<string, string>) {
```

Comment:
`preferLocal` -> `useLocal`

Can you describe the parameter? What's it a map of?